elements/rh-subnav/demo/rh-subnav.html

```diff
@@ -14,15 +14,30 @@ <h2>Default</h2>
   </rh-subnav>
 </div>
 
-<rh-context-provider class="example padded" color-palette="lighter">
-  <h2>In a container with padding</h2>
-  <rh-subnav>
+<div class="example">
+  <rh-context-provider class="padded" color-palette="lighter">
+    <h2>In a container with padding</h2>
+    <rh-subnav>
+      <a href="#">Users</a>
+      <a href="#">Containers</a>
+      <a href="#">Databases</a>
+      <a href="#" active>Servers</a>
+      <a href="#">System</a>
+      <a href="#"><pf-icon slot="icon" icon="database" set="fas" size="sm"></pf-icon> Network</a>
+      <a href="#">Cloud</a>
+    </rh-subnav>
+  </rh-context-provider>
+</div>
+
+<div class="example">
+  <h2>Dynamic activation</h2>
+  <rh-subnav id="demo">
```

Comment:
@nikkimk or @hellogreg PTAL and resolve this conversation to automerge if this passes user testing